tests/controllers/test_api06_changeset.py

```diff
@@ -93,3 +93,18 @@ async def test_changeset_crud(client: AsyncClient):
     assert '@min_lon' not in changeset
     assert '@max_lon' not in changeset
     assert changeset['@changes_count'] == 0
+
+
+async def test_changesets_unathorized_get_request(client: AsyncClient):
+    assert LEGACY_HIGH_PRECISION_TIME
+
+    r = await client.get('/api/0.6/changesets')
+    assert r.is_success, r.text
+
+    changesets: dict = XMLToDict.parse(r.content)['osm']
+    assert changesets['@version'] == 0.6
+    assert changesets['@generator'] == 'OpenStreetMap-NG'
+    assert changesets['@copyright'] == 'OpenStreetMap contributors'
+    assert changesets['@attribution'] == 'https://www.openstreetmap.org/copyright'
+    assert changesets['@license'] == 'https://opendatacommons.org/licenses/odbl/1-0/'
+    assert changesets['changeset'] is not None
```

Comment:
Most of it is not needed here:

```suggestion
    r = await client.get('/api/0.6/changesets')
    assert r.is_success, r.text

    changesets: dict = XMLToDict.parse(r.content)['osm']
    assert changesets['changeset'] is not None
```

LEGACY_HIGH_PRECISION_TIME ensures that timestamps are millisecond precision, and not just second, when making requests to API 0.6. This makes testing easier but it's not needed here.